zivid_camera/src/capture_settings_controller.cpp

```diff
@@ -12,28 +12,27 @@
 
 namespace zivid_camera
 {
-template <typename ConfigType>
+template <typename ConfigType, typename ZividSettings>
 class ConfigDRServer
 {
 public:
-  template <typename ZividSettings>
-  ConfigDRServer(const std::string& name, ros::NodeHandle& nh, const ZividSettings& defaultSettings)
+  ConfigDRServer(const std::string& name, ros::NodeHandle& nh, const Zivid::Camera& camera)
     : name_(name), dr_server_(dr_server_mutex_, ros::NodeHandle(nh, name_)), config_(ConfigType::__getDefault__())
   {
     static_assert(std::is_same_v<ZividSettings, Zivid::Settings> || std::is_same_v<ZividSettings, Zivid::Settings2D> ||
                   std::is_same_v<ZividSettings, Zivid::Settings::Acquisition> ||
                   std::is_same_v<ZividSettings, Zivid::Settings2D::Acquisition>);
 
-    const auto config_min = zividSettingsToMinConfig<ConfigType>(defaultSettings);
+    const auto config_min = zividSettingsMinConfig<ConfigType, ZividSettings>(camera);
     dr_server_.setConfigMin(config_min);
 
-    const auto config_max = zividSettingsToMaxConfig<ConfigType>(defaultSettings);
+    const auto config_max = zividSettingsMaxConfig<ConfigType, ZividSettings>(camera);
```

Comment:
Nice :+1: 
I am getting the correct default values with Zivid One now:
![image](https://user-images.githubusercontent.com/21991610/90402352-9c9a4800-e09f-11ea-9d5d-489266fd5d70.png)